python/datamodel/carpedestrian/datamodel.py

```diff
@@ -34,7 +34,7 @@ def ID(self):
     def ID(self, value):
         self._ID = value
 
-    _Position = Vector3(0, 0, 0)
+    _Position = Vector3(0,0,0)
```

Comment:
These shouldn't really exist.. It's an ugly trick, they should be removed and set in init instead. same applies to all other _<variable>